host_core/test/host_core/actors_test.exs

```diff
@@ -16,8 +16,75 @@ defmodule HostCore.ActorsTest do
   @kvcounter_key "MCFMFDWFHGKELOXPCNCDXKK5OFLHBVEWRAOXR5JSQUD2TOFRE3DFPM7E"
   @httpserver_link "default"
   @httpserver_contract "wasmcloud:httpserver"
+  @echo_old_oci_reference "wasmcloud.azurecr.io/echo:0.2.0"
   @echo_oci_reference "wasmcloud.azurecr.io/echo:0.2.1"
 
+  test "live update same revision fails" do
+    {:ok, _pid} = HostCore.Actors.ActorSupervisor.start_actor_from_oci(@echo_oci_reference)
+
+    on_exit(fn ->
+      HostCore.Actors.ActorSupervisor.terminate_actor(@echo_oci_key, 1)
+    end)
+
+    assert :error == HostCore.Actors.ActorSupervisor.live_update(@echo_oci_reference)
+  end
+
+  test "live update with new revision succeeds" do
+    {:ok, _pid} = HostCore.Actors.ActorSupervisor.start_actor_from_oci(@echo_old_oci_reference)
+
+    assert :ok == HostCore.Actors.ActorSupervisor.live_update(@echo_oci_reference)
```

Comment:
Thoughts on also including a scale test? e.g. start 5-10 instances of echo, live update, and then assert ok, check count, and ensure claims are of the updated revision?

Reply:
I'll be adding scale tests after we refactor the control interface to use scale operations rather than start/stop

Reply:
Sounds good to me 👍🏻